### PR TITLE
fix: Add missing documentation recommendations [DHIS2-13247]

### PR DESCRIPTION
**Pull request by maikelarabori**

### CODE REVIEW

File: src/developer/web-api/analytics.md [hunk 1]
```diff
@@ -181,6 +181,33 @@ the pagination parameter `paging=true` to the URL.
...
+
+In the example above, the client will receive back all the *Categories* that has `Data dimension` set to `true`.
+The *Categories*, in this case, will be the ones associated (through data sets and category combos) with the data element `fbfJHSPpUQD`.
+In addition, all *Organization Unit Group Set* that have `Data dimension` set to `true` will also (and always) be returned as part of the response.
+
+In other words, in a more technical approach, behind the scenes it does:
+
+1) For each data element (`dx`) provided in the URL as dimension do
+
+   `dx` -> GET category combos -> GET categories -> CHECK if `Data dimension` is true -> THEN include the `Category` in the response list.
```
Comment (larshelge) [inline src/developer/web-api/analytics.md]: I am wondering if this goes a bit too much into the inner details/workings of the API. Could we explain it a bit simpler and more from an external perspective perhaps @maikelarabori ?

Comment (Philip-Larsen-Donnelly) [inline src/developer/web-api/analytics.md]: @larshelge how about the alternative?
```suggestion
In the example above, the client will receive back all the *Categories* that are configured as `Data dimension`s and associated (through data sets and category combos) with the data element `fbfJHSPpUQD`.
In addition, all *Organization Unit Group Set*s that are configured as `Data dimension`s will also (and always) be returned as part of the response.

```

File: src/developer/web-api/analytics.md [hunk 2]
```diff
@@ -181,6 +181,33 @@ the pagination parameter `paging=true` to the URL.
...
+
+   `dx` -> GET data sets -> GET category combos -> GET categories -> CHECK `Data dimension` is true ->  THEN include the `Category` in the response list.
+
+2) Find all *Organization Unit Group Set* which has `Data dimension` set to `true` -> THEN include the *Organization Unit Group Set* in the response list.
```
Comment (Philip-Larsen-Donnelly) [inline src/developer/web-api/analytics.md]: Delete the remaining part as part of the simplification
```suggestion
```

File: src/developer/web-api/analytics.md (hunk 1 above)
Comment (Philip-Larsen-Donnelly) [reply src/developer/web-api/analytics.md]: Added another comment with alternative.

Comment (maikelarabori) [reply src/developer/web-api/analytics.md]: Thanks @larshelge / @Philip-Larsen-Donnelly.
We can remove the technical details. I'm ok with that.

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**